docs/source/user_guide/gui.md

```diff
@@ -45,44 +45,51 @@ Below, we'll explain how to do this.
 Though this is not strictly necessary, it is usually informative to
 view the keypoints overlaid on a background that provides
 some spatial context. You can either [load the video](target-load-video)
-corresponding to the dataset, or a [single image](target-load-frame),
-e.g., a still frame derived from that video.
-You can do this by dragging and dropping the corresponding file onto the
+corresponding to the dataset, or [load a single image](target-load-frame) (e.g., a still frame derived from that video).
+You can do this by dragging and dropping the corresponding video or image file onto the
 `napari` window or by using the `File > Open File(s)` menu option.
-Please read the following sections for detailed information
-and some important considerations.
+
 
 (target-load-video)=
 ### Load a video
 
-When trying to load a video file into `napari`, you will be prompted
+Upon loading a video file into `napari`, you will be prompted
 via a pop-up dialog to select the reader.
 Choose the `video` reader—corresponding to the
 [`napari-video`](https://github.com/janclemenslab/napari-video)
 plugin—and click `OK`. You can optionally select to remember this reader
 for all files with the same extension.
 
+(target-widget-video-reader)=
```

Comment:
Many of the image target are never cross-referenced. For explicit targets, my policy would be to add them on an as-needed basis, i.e. create the target the first time you need to reference it. This means that only one image should retain its target.